### PR TITLE
mpdas: init at 0.4.4

### PR DESCRIPTION
**Pull request by taketwo**
- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by yegortimoshenko** [comment]
Builds :+1:

File: pkgs/tools/audio/mpdas/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  buildInputs = [ mpd_clientlib curl ];
+
+  makeFlags = "DESTDIR= PREFIX=$(out)";
+
+  meta = with stdenv.lib; {
+    description = "Music Player Daemon AudioScrobbler";
+    homepage = http://50hz.ws/mpdas/;
+    license = licenses.bsd3;
+    maintainers = [ maintainers.taketwo ];
```
Comment (yegortimoshenko) [inline pkgs/tools/audio/mpdas/default.nix]: You need to add yourself to `lib/maintainers.nix` to do that:
https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix

File: pkgs/tools/audio/mpdas/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pkgconfig, mpd_clientlib, curl }:
+
+stdenv.mkDerivation rec {
+  name = "mpdas-${version}";
+  version = "0.4.4";
+
+  src = fetchFromGitHub {
+    owner = "hrkfdn";
+    repo = "mpdas";
+    rev = "${version}";
```
Comment (yegortimoshenko) [inline pkgs/tools/audio/mpdas/default.nix]: Nitpick: could be just `rev = version;`

File: pkgs/tools/audio/mpdas/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    sha256 = "1i6i36jd582y3nm5plcrswqljf528hd23whp8zw06hwqnsgca5b6";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildInputs = [ mpd_clientlib curl ];
+
+  makeFlags = "DESTDIR= PREFIX=$(out)";
```
Comment (yegortimoshenko) [inline pkgs/tools/audio/mpdas/default.nix]: Nitpick: lists are more easily overridable and semantically better:

```
makeFlags = [ "DESTDIR=" "PREFIX=$(out)" ];
```

**Review by lukateras — CHANGES_REQUESTED** [review]
Builds! :+1: But `meta.maintainers` should be fixed.

**Comment by taketwo** [comment]
I noticed that one of the Travis builds failed with:

```
error: attribute ‘taketwo’ missing, at /home/travis/build/NixOS/nixpkgs/pkgs/tools/audio/mpdas/default.nix:24:21
```

This is from the maintainer property in meta. Seems like I should add myself to [maintainers.nix](https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix), right? Should I make an extra commit or amend this one?

**Comment by yegortimoshenko** [comment]
Changes are typically amended, but I'm not sure about the policy on `lib/maintainers.nix` specifically. See file history: https://github.com/NixOS/nixpkgs/commits/master/lib/maintainers.nix

**Review by lukateras — APPROVED** [review]

**Comment by taketwo** [comment]
@yegortimoshenko Thanks for nitpicking :) Should be good to go now. From the history it seems like maintainer updates are usually combined with package updates, so I put it in the same commit.

**Comment by Mic92** [comment]
The program currently looks in $out/etc for configuration. `/etc` might be a better default.

```console
$ mpdas
(17:37:41) [INFO] Cannot parse config file (/nix/store/xcfmrnnqfi50hiaymxixcfvivdfqnlkw-mpdas-0.4.4/etc/mpdasrc).
(17:37:41) [ERROR] AudioScrobbler username or password not set
$ strings $(which mpdas) | grep etc
/nix/store/xcfmrnnqfi50hiaymxixcfvivdfqnlkw-mpdas-0.4.4/etc
```

**Comment by grahamc** [comment]
@GrahamcOfBorg mpdas

**Comment by yegortimoshenko** [comment]
Yeah, `/etc` is probably better as Nix store is read-only. There is a `CONFIG` variable for that:

```
makeFlags = [ "CONFIG=/etc" "DESTDIR=" "PREFIX=$(out)" ];
```

**Comment by taketwo** [comment]
I was about to push a fix, but @Mic92 was faster! Thank you guys for quick responses and suggestions.

**Comment by taketwo** [comment]
BTW, can you please clarify when this package will become installable with `nix-env -i mpdas`?

**Comment by Mic92** [comment]
On unstable with the next channel update: https://howoldis.herokuapp.com/

**Comment by Mic92** [comment]
Can be also monitored here: https://hydra.nixos.org/jobset/nixpkgs/trunk/evals

**Comment by taketwo** [comment]
Thanks for the pointers. If I understood correctly, no particular schedule, maybe tomorrow, maybe in a week, as soon as certain important failing packages turn green.

**Comment by Mic92** [comment]
@taketwo hydra has build it already, but the complete evaluation is not finished yet: https://hydra.nixos.org/eval/1408888?filter=mpdas&compare=1408845&full=

**Pull request merged**